README.md

```diff
@@ -159,7 +159,7 @@ const chromeless = new Chromeless({
 - [`exists(selector: string)`](docs/api.md#api-exists)
 - [`screenshot()`](docs/api.md#api-screenshot)
 - [`pdf(options?: PdfOptions)`](docs/api.md#api-pdf)
-- [`getHtml()`](docs/api.md#api-gethtml)
+- [`html()`](docs/api.md#api-html)
```

Comment:
@seangransee glad that you like it! The maintainers had a conversation about API design goals yesterday where we decided on this naming convention. I'll share more about the design goals soon in a docs/api-design.md linked to via CONTRIBUTING.md.